src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -629,6 +659,10 @@
 long-summary: |
     For Linux VMs, disabling encryption is only permitted on data volumes.
     For Windows VMS, disabling encryption is permitted on both OS and data volumes.
+examples:
+  - name: Disable disk encryption on the OS disk and/or data disks. (autogenerated)
```

Comment:
This description is wrong. It is clearly disabling encryption only on data disks...

Reply:
I agree, we will keep this feedback in mind for getting better descriptions in future examples.